src/lucky_env.cr

```diff
@@ -5,6 +5,12 @@ require "./lucky_env/*"
 module LuckyEnv
   VERSION = "0.1.2"
 
+  macro add_env(name)
+    def LuckyEnv.{{ name.id }}?
+      environment == {{ name.id.stringify }}
+    end
+  end
+
```

Comment:
Oh nice! Ok, this is awesome. This means in a generated app, we can have the `config/env.cr` file be some comments on how to use this to configure additional envs plus document using these methods 🥳  Good call ❤️